core/arch/arm/kernel/generic_boot.c

```diff
@@ -534,6 +534,19 @@ void *get_external_dt(void)
 
 static void release_external_dt(void)
 {
+	int ret;
+
+	/*
+	 * Close DT after all drivers load
+	 * It allows drivers load to modify the DT
+	 */
+	ret = fdt_pack(external_dt.blob);
```

Comment:
What if there is no external DTB. Original code did not call `fdt_pack()` but it will now since it's called from `release_external_dt()`.